web_poet/mixins.py

```diff
@@ -97,7 +116,10 @@ def url(self) -> str:
     @property
     def html(self) -> str:
         """Shortcut to HTML Response's content."""
-        return self.response.text
+        try:
+            return self.response.html  # type: ignore[union-attr]
+        except AttributeError:
+            return self.response.text  # type: ignore[union-attr]
```

Comment:
The PR looks good overall, but I'm not a fan of this approach. I can see two alternatives:

1. BrowserResponse has _selector_input, which "knows" that html should be used, not text. Maybe we can use something like this here. But I'm not sure how to do it cleanly.
2. Add text attribute to BrowserResponse. It seems it would allow to drop some code, including code in AnyResponse implementation.

Currently I'm +0.5 to follow (2). Thoughts?

Reply:
> Add text attribute to BrowserResponse.

With the same content as the `text` attribute (so `BrowserHtml`), or with just `str` (though `BrowserHtml` is a subclass of `str`)?

> It seems it would allow to drop some code, including code in AnyResponse implementation.

Indeed. I think it makes sense.